src/components/CharacterCardList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, ReactNode } from 'react';
+import { Character } from '../types';
+
+interface CharacterCardListProps {
+  characters: Character[];
+  isLoading: boolean;
+}
+
+class CharacterCardList extends Component<CharacterCardListProps> {
+  render(): ReactNode {
+    const { characters, isLoading } = this.props;
+    return (
+      <>
+        {isLoading ? (
+          <div className="d-flex align-items-center justify-content-around my-3">
+            <strong>Loading...</strong>
+            <div className="spinner-border ml-auto" role="status" aria-hidden="true"></div>
+          </div>
+        ) : null}
+
+        <div className="d-flex flex-wrap justify-content-around">
+          {characters.length > 0 || isLoading ? (
+            characters.map((character) => <CharacterCard key={character.id} character={character} />)
+          ) : (
+            <p>Try search something else (Butter Robot, Abradolf Lincler, Rick, Morty) </p>
+          )}
+        </div>
+      </>
+    );
+  }
+}
+
+interface CharacterCardProps {
+  character: Character;
+}
+
+class CharacterCard extends Component<CharacterCardProps> {
```

Comment:
2 classes in 1 file? I guess it is not a best practice.
We also need to mention the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) and other [SOLID](https://en.wikipedia.org/wiki/SOLID) principles Please, check [General requirements for all the tasks](https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/tasks/README.md#general-requirements-for-all-the-tasks)